libcaliptra/examples/hwmodel/interface.c

```diff
@@ -24,14 +23,14 @@
 
 struct caliptra_model_init_params init_params;
 struct caliptra_fuses fuses = {0};
+
+__attribute__((section("VPK_HASH"))) uint8_t vpk_hash[48];
+__attribute__((section("OPK_HASH"))) uint8_t opk_hash[48];
```

Comment:
It seems like it might be good to have the option to do this at runtime or build time.

If the SoC FW which uses libcaliptra is always bundled with a specific Caliptra image then this works. But if not, I think it is still desirable to extract they keys at runtime. wdyt?

Reply:
I think that's outside our scope? How that key digest is delivered to the firmware is going to be device and implementer specific (i.e. the SoC may do that digest with whatever library/hw it has,) and this way is simply a reflection on the "hwmodel" behavior.

We might need an "Developer's Guide" that explains everything.

Reply:
Also, IIRC the key digest is supposed to be burned in at manufacture, so most of the time you won't have the digest step at all.

Reply:
Ok, that makes sense. Agreed that if the SoC vendor needs to do something different they can handle that themselves.